bot.properties.example

```diff
@@ -12,3 +12,7 @@ github-token: github_oauth_token
 sentry-dsn: https://xxx@xxx.ingest.sentry.io/xxx
 sentry-env: production
 ocr-path: /usr/share/tesseract-ocr/4.00/tessdata
+algolia-application-id: 0DTHI9QFCH
+algolia-search-api-key: 3cc0567f76d2ed3ffdb4cc94f0ac9815
```

Comment:
Should we not use a separate API key if the portal lets us see search trends per key? Might be useful to see what people are looking for on discord vs the wiki its self.

Reply:
It doesn't let you do that, and it doesn't let you generate multiple API keys. You only get one public API key, and one write API key.